Core/GDCore/Extensions/Builtin/WindowExtension.cpp

```diff
@@ -38,6 +38,17 @@ void GD_CORE_API BuiltinExtensionsImplementer::ImplementsWindowExtension(
                     true)
       .SetDefaultValue("yes");
 
+  extension
+      .AddCondition(
```

Comment:
it looks like some commit from another pr somehow bleeded  over this one

Reply:
I think that's a GitHub issue, try to close this PR and open another one. That should do the trick. It's because master got merged in this PR.

Reply:
Just checking in on this, is the new PR going to be a big issue for progress, @blurymind ?

Reply:
Normally it's nothing at all, should just give more clarity before merging.